img/o/omny.info.svg

```diff
@@ -0,0 +1 @@
 .8-.2 3.1-.7 5-1 13.8-2.3 40.3-10.3 60.4-18.2 17-6.7 42.6-19.3 51.6-25.3 1.4-.9 7-4.6 12.5-8.2 30-19.5 61.5-49.6 83.6-79.8 19.5-26.7 34.7-56.1 46-89 6.5-18.9 14.2-52.2 15.5-67.4.3-3.3.7-6.2.9-6.6 1-1.6 1.9-24 1.9-44.5 0-20.7-.6-33.3-1.9-43.5-.2-1.9-.7-5.7-1.1-8.4-.3-2.7-.7-5.6-.9-6.5-.1-.9-.6-3.6-1-6.1-.8-5-.7-4.6-3-15a337.1 337.1 0 0 0-66.4-140c-14.9-18.6-40.6-43-61.1-58.1-47.8-35.3-110.9-59.6-170-65.5-3.6-.3-6.8-.8-7.1-1-.3-.2-8.4-.6-18-.9-9.6-.4-17.9-.8-18.6-1-.6-.2-1.3-.2-1.5 0zm23.2 47c5.8.3 11.6.8 13 1 1.4.3 4.5.7 7 1 38.1 4.7 81.1 19.4 114.5 39.3 51.4 30.6 93 77.4 116.7 131.2 7.9 18.1 15.2 39.8 17.5 51.9.2 1.4.6 2.7.8 3 .5.9 4.4 22.9 5 28.6.3 2.7.7 6.6 1 8.5 2.3 19.5 2.3 53.1 0 71-.3 1.9-.7 5.5-1 8-.7 5-1.1 7.5-3.1 19.5-1.7 10.1-9 35.6-13.9 48.5-5.9 15.4-13.9 31.9-22.8 46.7-39.1 65.3-103.8 113-178.7 131.8-7.4 1.9-15.1 3.7-17 4-1.9.3-6.2 1-9.4 1.6-3.3.6-7.8 1.3-10 1.5-2.3.3-5.2.7-6.6.9-12.3 2.2-54.2 2.2-69.4 0-1.7-.2-5.1-.7-7.6-.9-28.2-3.2-64-13.9-92.5-27.7-42.6-20.5-80.5-51.9-107.1-88.4-31.5-43.3-49.9-91-55.8-145-1.5-13.7-1.8-18.7-1.8-35.4 0-17.2.5-28.1 1.8-37.2.2-2 .7-5.6.9-8 .3-2.4.7-5.5 1-6.9.2-1.4.7-4.1 1-6 1.2-7.9 4.7-22.6 7.6-32.3 1.6-5.6 3.1-10.7 3.3-11.2 2.1-7 10.5-27.4 15.3-37 9.3-18.5 22.8-40.4 29.8-48.1.5-.7 4.2-5.1 8-9.8 8.6-10.7 28.1-29.9 39.5-38.9 35.6-28.3 77.1-48.2 121.2-58.1a275 275 0 0 1 15.3-3.1c2.2-.4 4.7-.8 5.5-1 .8-.2 4.2-.7 7.5-1.1 3.3-.3 6.2-.7 6.5-.9.3-.2 7.1-.7 15-1.1 8-.3 14.6-.7 14.7-.8.4-.3 17.9.2 27.3.9z"/><path d="M330.31 86.15c-.2.1-3.8.6-8.1 1-4.3.3-8.8.8-10 1-22.9 3.9-26.9 4.8-43.2 9.8-17.1 5.3-41.6 16.2-54.3 24.3-2.9 1.9-5.4 3.4-5.7 3.4-1 0-18.6 12.7-26 18.9a250.78 250.78 0 0 0-73.5 100.6c-3.1 7.8-7.7 20.8-8.5 24.3-.3.9-1.4 5.3-2.5 9.7-1.1 4.4-2.2 9.1-2.5 10.5-.2 1.4-.7 3.6-.9 5-3.4 17.2-4.4 31.8-4.1 56 .4 34.6 4.1 56.7 14.2 86 7.9 23 20.1 46.6 34.4 66.3 5.2 7 5.9 8 12.3 15.6 4.9 5.6 23.8 24.5 28.6 28.5 25.9 21.2 54 36.9 84 46.9 30.1 10 54.9 13.7 89 13.2 12.4-.1 23.4-.5 24.5-.8 1.1-.2 4.7-.8 8-1.1 3.3-.4 6.7-.9 7.5-1.1.8-.2 3.3-.6 5.5-1 6.2-.9 16.1-3.3 27.2-6.6 71.5-21.2 129.3-71.8 159.7-139.6 5-11.1 11.8-30.9 14.3-41.3.8-3.6 1.7-7.4 2-8.5.6-2.3 2-10.4 2.8-15.5.3-2 .7-4.5.9-5.7.3-1.2.8-5.3 1.1-9 .4-3.7.8-7.9 1.1-9.3.6-3.5.5-33-.1-41.5-.4-6.3-1.3-13.5-3-25-3.4-22.6-11.8-49.6-22.2-71.2-30.6-63.5-85.1-110.7-152.3-131.8-9-2.9-25.8-7-32-8-2.5-.4-5.4-.9-6.5-1.2-7.9-1.7-27.9-3.1-43.2-3.1-10 0-18.3.1-18.5.3zm46.2 46.9c1.9.3 6.2 1.1 9.5 1.7 41.2 7.2 80 27.5 110.5 57.9 34.6 34.6 56.6 83.1 60.7 134 1.5 18.3.6 42.2-2.2 59.2-1.2 7-1.3 7.6-4 17.8-8.6 33.6-25.4 65.4-46.2 88-2.3 2.5-5.4 5.9-7 7.6-3.7 4.1-16 15.3-19.6 17.9-1.6 1.1-5.6 4-9 6.5-22 16.1-52.4 29.2-79.7 34.3-4.7.9-10.1 1.9-12 2.3-5.7 1-46 1.3-52 .3-3-.5-7.5-1.1-10-1.5-2.5-.3-5.3-.8-6.2-1-1-.3-2.8-.7-4-.9-6.5-1.2-18.5-4.7-27.3-8-62.1-23.3-107.7-72-127.4-136-2.8-9.3-6.3-24.7-7.5-34-3.1-22.1-2.6-53.7 1.1-72l.9-5c1.4-9.3 8.5-33 13.1-43.5 17.4-39.7 43.6-70.9 79-94.1 11.1-7.2 34-18.3 46.3-22.3 11.7-3.8 30.6-8.2 38.9-9 3-.3 5.6-.8 5.9-1 .6-.7 43.9.1 48.2.8z"/></svg>
```

Comment:
Converting to path to circle elements makes the file smaller. Frustratingly my editor added transform attributes to the elements which I'm unable to revert :roll_eyes: 
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke="#000" stroke-linejoin="round" viewBox="0 0 26.67 26.46"><circle cx="52.61" cy="52.61" r="52.61" stroke-width="8.26" transform="translate(1.07 .96) scale(.2332)"/><circle cx="52.61" cy="52.61" r="52.61" stroke-width="10.74" transform="translate(3.9 3.8) scale(.1793)"/></svg>
```